test/feed.test.js

```diff
@@ -91,5 +124,62 @@ describe('Post data class tests', () => {
       await feed.setDelayed(60);
       expect(feed.isDelayed()).resolves.toBe(true);
     });
+
+    test('Updated feed should be different from data', async () => {
+      const feed = new Feed(data.author, data.url, data.user, data.link, null, null);
+      feed.author = 'Author Post';
+      feed.url = 'https://modified.user.feed.com/feed.rss';
+      await feed.update();
+      const modifiedFeed = await Feed.byId(feed.id);
+      // modifiedFeed.id should be different from feed.id as feed hasn't been reassigned to updated feed.id value
+      expect(modifiedFeed.id).not.toBe(feed.id);
+      expect(modifiedFeed.id).not.toBe(createFeed.id);
+      expect(modifiedFeed.url).not.toBe(createFeed.url);
+      expect(modifiedFeed.author).not.toBe(createFeed.author);
+      expect(modifiedFeed.url).toBe(feed.url);
+      expect(modifiedFeed.author).toBe(feed.author);
+      expect(modifiedFeed.id).toBe(urlToId(feed.url));
+    });
+
+    test('Removing feeds should also remove posts', async () => {
+      const feed = await Feed.byId(await Feed.create(data));
+      postData1.feed = feed;
+      postData2.feed = feed;
+      await Promise.all([
+        Post.create(postData1),
+        Post.create(postData2),
```

Comment:
These should probably be using `createFromArticle()` and we can probably delete `Post.create()` (in a follow-up PR if you want to file that).